src/Spectre.libStatistics.Tests/DifferentiatingFeaturesEstimatorTest.cpp

```diff
@@ -72,7 +72,7 @@ TEST(DifferentiatingFeaturesEstimator, estimates_without_throw_for_valid_dataset
 {
     const ValuesHomogeneityEstimatorMock mock;
     DifferentiatingFeaturesEstimator estimator(mock);
-    EXPECT_CALL(mock, Compare(_, _)).WillRepeatedly(Return(index));
+    EXPECT_CALL(mock, Compare(_, _)).WillRepeatedly(Return(_index));
```

Comment:
Is there a scope conflict of variable naming?

Reply:
Yeah, @mg6 reported some problems MSVC was crying about w.r.t. `index` identifier.

Reply:
Exactly; `index` identifier clashed with GSL one, preventing a successful build.